facade/src/main/java/org/jboss/pnc/facade/providers/DeliverableAnalyzerReportProviderImpl.java

```diff
@@ -74,7 +102,7 @@ public Page<AnalyzedArtifact> getAnalyzedArtifacts(
             String query,
             String sort,
             String id) {
-        Base32LongID entityId = mapper.getIdMapper().toEntity(id);
+        Base32LongID entityId = transformToEntityId(id);
```

Comment:
We have parseId()

Reply:
Nice catch! [1e6273a](https://github.com/project-ncl/pnc/pull/3840/commits/1e6273a4e7093662711148bad66023a2233d4fdc)